forge/test/operators/matmul/test_matmul_fuse_bias.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import numpy as np
+
+import torch
+from torch import nn
+
+import forge
+from forge.op.eval.common import compare_with_golden_pcc
+
+import os
+
+@pytest.mark.parametrize("dims", [
```

Comment:
These are more shapes as it seems :)) Also make sure to use formatted from pre-commit hooks :))

Reply:
fixed